cli/heartbeat.js

```diff
@@ -211,10 +212,96 @@ async function update(status, extra) {
   return info;
 }
 
+async function getOverheatedThresholds() {
+  const uname = await getShellOutput("uname -m");
+  let temperatureThreshold = null;
+  let timeThreshold = null;
+  switch (uname) {
+    case "aarch64": {
+      const boardName = await getShellOutput("awk -F= '/BOARD=/ {print $2}' /etc/firewalla-release");
```

Comment:
cache the local threshold per process life time, no need to query per interval

Reply:
Correct me if I am wrong about `setInterval` call. 

The thresholds are calculated inside `getOverheatedThresholds()`, which is only invoked once inside `setInterval()` call. And later run of `monitorTemperature` will just use the same passed-in value, I suppose.

Reply:
Go with the way of global variable